external/libcxx-test/tc_common.h

```diff
@@ -0,0 +1,154 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __EXTERNAL_LIBCXX_TEST_TC_COMMON_H
```

Comment:
Oh. You copied content from original tc_common.h. But could you let me know the reason?

Reply:
Yes, tc_common.h in not common place. I could include it via <> or <tinyara/*>. If we can directly include , they copying this file is not required.

Reply:
This module is not in testcase. Originally only testcase which is in testcase folder needs tc_common.h. This is an exception of testcase so how about use CFLAGS to include original tc_common.h?

Reply:
Ok. I will include it using CFLAGS